src/utils/history-sort.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var _ = require('underscore');
+var dateStringified = require('./date-stringify');
+
+module.exports = function(history) {
+  var result = [];
+  
+  _.each(history.components, function(singleComponent, name) {
```

Comment:
I guess histories could grows quite long in time.. then for loops instead of nested each mill make quite a difference, we could also probably create a sorted array as we push them instead of having to loop through it again with sortBy. What you think?